src/OpenTelemetry.Api.ProviderBuilderExtensions/OpenTelemetry.Api.ProviderBuilderExtensions.csproj

```
@@ -1,8 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworks)</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>$(DefaultTargetFrameworks)</TargetFrameworks>
```

This line, as it is currently written, isn't doing anything. It is just setting a property to itself.